### PR TITLE
LAM-972 Refactor/use recyclers instead of stackers

### PR DESCRIPTION
**Pull request by RafaelTaranto**

### CODE REVIEW

File: lib/cash-in/cash-in-atomic.js [hunk 1]
```diff
@@ -50,17 +50,17 @@ function insertNewBills (t, billRows, machineTx) {
...
-    getBillsByDestination('stacker2f').length,
-    getBillsByDestination('stacker2r').length,
-    getBillsByDestination('stacker3f').length,
-    getBillsByDestination('stacker3r').length
+    getBillsByDestination('recycler1').length,
+    getBillsByDestination('recycler2').length,
+    getBillsByDestination('recycler3').length,
+    getBillsByDestination('recycler4').length,
+    getBillsByDestination('recycler5').length,
+    getBillsByDestination('recycler6').length
```
Comment (siiky) [inline lib/cash-in/cash-in-atomic.js]: I propose ditching `getBillsByDestination()` and instead:

```js
  const billsByDestination = _.countBy(_.get(['distination_unit']) ,dbBills)

  // ...

  return t.none(sql2, [
    deviceID,
    _.defaultTo(0, billsByDestination.recycler1),
    _.defaultTo(0, billsByDestination.recycler2),
    _.defaultTo(0, billsByDestination.recycler3),
    _.defaultTo(0, billsByDestination.recycler4),
    _.defaultTo(0, billsByDestination.recycler5),
    _.defaultTo(0, billsByDestination.recycler6)
  ])
  // ...
```

File: new-lamassu-admin/src/pages/Maintenance/Wizard/WizardStep.js [hunk 2]
```diff
@@ -118,21 +118,19 @@ const cassetesArtworks = (step, numberOfCassettes, numberOfStackers) => {
...
   const cassetteStepsStart = CASHBOX_STEP + 1
   if (step < cassetteStepsStart + numberOfCassettes)
     return {
       name: `cassette${step - cassetteStepsStart + 1}`,
       category: 'cassette'
     }
-  const stackerStepsStart = CASHBOX_STEP + numberOfCassettes + 1
-  if (step < stackerStepsStart + numberOfStackers * 2)
+  const recyclerStepsStart = CASHBOX_STEP + numberOfCassettes + 1
+  if (step < recyclerStepsStart + numberOfRecyclers * 2)
```
Comment (siiky) [inline new-lamassu-admin/src/pages/Maintenance/Wizard/WizardStep.js]: `s/numberOfRecyclers * 2/numberOfRecyclers/` ?

File: new-lamassu-admin/src/pages/Notifications/sections/FiatBalanceAlerts.js [hunk 3]
```diff
@@ -183,19 +183,22 @@ const FiatBalance = ({ section, min = 0, max = 100, fieldWidth = 80 }) => {
                         labelClassName={classes.cashboxLabel}
                         emptyPartClassName={classes.cashboxEmptyPart}
                         percent={
-                          values[`fillingPercentageStacker${it + 1}f`] ??
-                          data[`stacker${it + 1}f`]
+                          values[
+                            `fillingPercentageRecycler${(it + 1) * 2 - 1}`
+                          ] ?? data[`recycler${(it + 1) * 2 - 1}f`]
```
Comment (siiky) [inline new-lamassu-admin/src/pages/Notifications/sections/FiatBalanceAlerts.js]: extra `f`

File: new-lamassu-admin/src/pages/Notifications/sections/FiatBalanceAlerts.js [hunk 4]
```diff
@@ -232,7 +237,7 @@ const FiatBalance = ({ section, min = 0, max = 100, fieldWidth = 80 }) => {
                     </div>
                   </>
                 ],
-                R.times(R.identity, maxNumberOfStackers)
+                R.times(R.identity, maxNumberOfRecyclers)
```
Comment (siiky) [inline new-lamassu-admin/src/pages/Notifications/sections/FiatBalanceAlerts.js]: This must be `maxNumberOfRecyclers / 2` or the steps must be halved.

File: lib/plugins.js [hunk 5]
```diff
@@ -817,74 +812,74 @@ function plugins (settings, deviceId) {
       }
       : null
     
-    const stacker1fAlert = device.numberOfStackers >= 1 && isUnitLow(device.cashUnits.stacker1f, getCashUnitCapacity(device.model, 'stacker'), notifications.fillingPercentageStacker1f)
+    const recycler1Alert = device.numberOfRecyclers >= 1 && isUnitLow(device.cashUnits.recycler1, getCashUnitCapacity(device.model, 'recycler'), notifications.fillingPercentageRecycler1)
       ? {
         code: 'LOW_RECYCLER_STACKER',
         cassette: 4,
```
Comment (siiky) [inline lib/plugins.js:818]: what are these `cassette: 4` fields?

**Review by siiky — CHANGES_REQUESTED** [review]

File: lib/plugins.js (hunk 5 above)
Comment (RafaelTaranto) [reply lib/plugins.js:818]: I'll look into it further on the notification specific card.

**Pull request merged**